recipe-client-addon/bootstrap.js

```diff
@@ -68,7 +70,13 @@ this.startup = function() {
   CleanupManager.addCleanupHandler(
     () => Preferences.ignore(PREF_LOGGING_LEVEL, LogManager.configure));
 
-  RecipeRunner.init();
+  // Initialize experiments first to avoid a race between initializing prefs
+  // and recipes rolling back pref changes when experiments end.
+  PreferenceExperiments.init().catch(err => {
+    log.error("Failed to initialize preference experiments:", err);
```

Comment:
I would've liked to test that `PreferenceExperiments.init` failing does not stop `RecipeRunner.init` from running, but I dunno how to test bootstrap stuff.